### PR TITLE
Sdk docs update

### PR DESCRIPTION
**Pull request by canhtrinh**

### CODE REVIEW

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 1]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+* Destination chains can be either an EVM chain or cosmos-based chain
+* When sending to an EVM chain, the asset will arrive as an ERC-20 asset
+
+Example acceptance criteria: 
+`I want to send axlUSDC on Osmosis and receive the equivalent ERC-20 version on Avalanche.`
+
+```tsx
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "osmosis-5",
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:122]: Could we use the CHAINS config object? (request from Milap)

```ts
const fromChain = CHAINS.TESTNET.OSMOSIS;
const toCHain = CHAINS.TESTNET.AVALANCHE;
```

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 2]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+##### 1. Send tokens from a Cosmos-based chain
+
+Transfer any Axelar-supported asset from a cosmos-based chain to a destination chain. 
+* Destination chains can be either an EVM chain or cosmos-based chain
+* When sending to an EVM chain, the asset will arrive as an ERC-20 asset
+
+Example acceptance criteria: 
+`I want to send axlUSDC on Osmosis and receive the equivalent ERC-20 version on Avalanche.`
+
+```tsx
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:119]: should be ts instead i think

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 3]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+* When sending to an EVM chain, the asset will arrive as an ERC-20 asset
+
+Example acceptance criteria: 
+`I want to send axlUSDC on Avalanche and receive axlUSDC on Osmosis.`
+
+```tsx
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "avalanche", 
+  toChain = "osmosis-5",
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:148]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 4]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
-  "kujira1tck82gz5v5rzc74hmf8j9vyjcs3nnnyc6dgc3c", // destination address
-  "uausdc" // denom of asset. See note (2) below
-);
 ```
 
-Notes:
+Example acceptance criteria: 
+`I want to send WAVAX on Polygon and receive AVAX on Avalanche.`
+
+```tsx
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:220]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 5]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+Also known as "wrap", transfer a wrapped native token from an EVM chain back to its home EVM chain
+* The destination chain must be the native chain on the wrapped native asset
+* You can specify an optional parameter in the configs for `GetDepositAddressParams` whether you want to receive the asset in the form of native tokens on the destination chain OR its ERC-20 equivalent. 
+
+Two scenarios below
+
+Example acceptance criteria: 
+`I want to send WAVAX on Polygon and receive WAVAX on Avalanche.`
 
 ```tsx
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:198]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 6]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+
+Also known as "wrap", transfer the native asset of an EVM chain to any destination chain.
+* Destination chains can be either an EVM chain or cosmos-based chain
+* When sending to an EVM chain, the asset will arrive as an ERC-20 "wrapped" asset
+* The only difference here vs. case (1) above is that the specified asset is the native asset of the source EVM chain. 
+
+Example acceptance criteria: 
+`I want to send AVAX on Avalanche and receive the equivalent ERC-20 version on Polygon.`
 
 ```tsx
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:171]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 7]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
-const sdk = new AxelarAssetTransfer({
-  environment: "testnet",
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "avalanche", 
+  toChain = "polygon",
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:175]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 8]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
+`I want to send WAVAX on Polygon and receive WAVAX on Avalanche.`
 
 ```tsx
-const sdk = new AxelarAssetTransfer({
-  environment: "testnet",
-  auth: "local",
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "polygon", 
+  toChain = "avalanche",
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:202]: same as above

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md [hunk 9]
```diff
@@ -81,55 +81,169 @@ async function main() {
...
 
-Notes:
+Example acceptance criteria: 
+`I want to send WAVAX on Polygon and receive AVAX on Avalanche.`
+
+```tsx
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "polygon", 
+  toChain = "avalanche",
```
Comment (vladwulf) [inline pages/dev/axelarjs-sdk/token-transfer-dep-addr.md:224]: same as above

**Review by vladwulf — APPROVED** [review]

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 1 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 3 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 4 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 6 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 8 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 9 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 2 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

File: pages/dev/axelarjs-sdk/token-transfer-dep-addr.md (hunk 5 above)
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/token-transfer-dep-addr.md]: done

**Pull request merged**